consensus/polybft/polybft.go

```diff
@@ -143,15 +143,15 @@ func GenesisPostHookFactory(config *chain.Chain, engineName string) func(txn *st
 			rootNativeERC20Token = polyBFTConfig.Bridge.RootNativeERC20Addr
 		}
 
-		if polyBFTConfig.MintableERC20Token {
+		if polyBFTConfig.MintableNativeToken {
```

Comment:
Just for the record:
native token in fact follows ERC20 standard, so it was not wrong as it was before.